Rocket.Chat/Controllers/Chat/ChatControllerMessageCellProtocol.swift

```diff
@@ -19,6 +20,33 @@ extension ChatViewController: ChatMessageCellProtocol {
         }
     }
 
+    func handleTapMessageCell(_ message: Message, view: UIView, recognizer: UIGestureRecognizer) {
+        guard let username = message.user?.name else { return }
+
+        func openDirectMessage() -> Bool {
+            guard let directMessageRoom = Realm.shared?.objects(Subscription.self).filter("name == '\(username)' && privateType == 'd'").first else { return false }
```

Comment:
Let's remove this Realm dependency. You can add this as a method in `Subscription` model for now.